spacy/cli/project/remote_storage.py

```diff
@@ -84,7 +85,20 @@ def pull(
                 with tarfile.open(tar_loc, mode=mode_string) as tar_file:
                     # This requires that the path is added correctly, relative
                     # to root. This is how we set things up in push()
-                    tar_file.extractall(self.root)
+                    def is_within_directory(directory, target):
```

Comment:
Let's add a reference to [CVE-2007-4559](https://github.com/advisories/GHSA-gw9q-c7gh-j9vm) in the comments maybe?